src/main/javascript/crypto/src/Authenticator.ts

```diff
@@ -163,7 +81,7 @@ export class Authenticator {
             logger(DEBUGLEVEL.MEDIUM,'ticked valid (signature OK)');
         } catch (e) {
             logger(DEBUGLEVEL.MEDIUM,'getUseTicket: ticket validation failed',e);
-            throw new Error("getUseTicket: ticked validation failed");
+            throw new Error("getUseTicket: ticked validation failed: " + e.message);
```

Comment:
we need some way to set DEBUGLEVEL, because it can't be set in config.ts, because its will be ready NPM package.
lets add some variable to the .env , so code can read and use it

Reply:
added commit, where I moved debug variable to the process.env.DISPLAY_DEBUG_LEVEL or window.DISPLAY_DEBUG_LEVEL so we can change debug level even for UMD builds